### DISCUSSION TITLE
Pedido

### DISCUSSION THREAD

**Discussion by hermesgamestv** [MrMadaraUchiha/MyCommand#8] [Q&A] [answered]
Gostaria que me ajudasse a criar um comando de /vote para meu server

**Comment by MrMadaraUchiha** [comment, marked as answer]
Claro aqui está um exemplo https://github.com/MrMadaraUchiha/MyCommand/blob/main/commands/vote_menu.yml

**Comment by hermesgamestv** [comment]
obrigado funcionou aqui

**Reply by MrMadaraUchiha** [reply]
Que bom fico feliz em ter ajudado!
